Dockerfile

```diff
@@ -39,4 +39,5 @@ RUN apt-get update && apt-get install docker-ce -y --no-install-recommends
 WORKDIR /var/lib/ocurrent
 ENTRYPOINT ["dumb-init", "/usr/local/bin/ocaml-ci-service"]
 ENV OCAMLRUNPARAM=a=2
+ENV DOCKER_CLI_EXPERIMENTAL=enabled
```

Comment:
What's this for?

Reply:
The image now does docker manifest pull to calculate opam vars (before it uses the pool), so the ocaml-ci image needs it enabled too.  I've added a comment to the Dockerfile in 2fd97826a0022d24cb3b85943f64354f1eb6a598